### ISSUE TITLE
Chat slowdown.

### ISSUE THREAD

**Issue by Klockchops** [codetorex/SteamMonsterLobby#3]
Chat slows down to a crawl after 5 lines or so. Don't know if we are ddosing it but it is not updating.

**Comment by codetorex** [comment]
Yeah too much connections slows the server down.Anyway, I will remove chat and put announcements screen instead .
